src/lib/Behat/PageObject/Page.php

```diff
@@ -107,7 +107,7 @@ public function getPageTitle(): string
      *
      * @return string
      */
-    public function getPageRoute(): string
+    protected function getCurrentRoute(): string
```

Comment:
As indicated in my original comment, the function `verifyRoute` should use this function:
https://github.com/ezsystems/ezplatform-admin-ui/blob/1.5/src/lib/Behat/PageObject/Page.php#L77

to make things consistent